cmd/cainjector/app/start.go

```diff
@@ -131,18 +134,20 @@ servers and webhook servers.`,
 }
 
 func (o InjectorControllerOptions) RunInjectorController(ctx context.Context) error {
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                        api.Scheme,
-		Namespace:                     o.Namespace,
-		LeaderElection:                o.LeaderElect,
-		LeaderElectionNamespace:       o.LeaderElectionNamespace,
-		LeaderElectionID:              "cert-manager-cainjector-leader-election",
-		LeaderElectionReleaseOnCancel: true,
-		LeaseDuration:                 &o.LeaseDuration,
-		RenewDeadline:                 &o.RenewDeadline,
-		RetryPeriod:                   &o.RetryPeriod,
-		MetricsBindAddress:            "0",
-	})
+	mgr, err := ctrl.NewManager(
+		util.RestConfigWithUserAgent(ctrl.GetConfigOrDie(), "ca-injector"),
```

Comment:
should this be `cainjector` instead? We don't refer to it as `ca-injector` anywhere else (including the leader election ID)